clients/drcachesim/tests/record_filter_invalidate_cpu.templatex

```diff
@@ -0,0 +1,24 @@
+Estimation of pi is 3.142425985001098
+
+Trace invariant checks passed
+
+Output .* entries from .* entries.
+
+Output format:
+
+<--record#-> <--instr#->: <---tid---> <record details>
+
+------------------------------------------------------------
+
+           1           0: +[0-9]+ <marker: version [0-9]>
+           2           0: +[0-9]+ <marker: filetype 0x[0-9a-f]*>
+           3           0: +[0-9]+ <marker: cache line size [0-9]*>
+           4           0: +[0-9]+ <marker: chunk instruction count [0-9]*>
+           5           0: +[0-9]+ <marker: page size [0-9]*>
+           6           0: +[0-9]+ <marker: timestamp [0-9]*>
+#ifdef X64
+           7           0: +[0-9]+ <marker: tid [0-9]* on core 18446744073709551615>
```

Comment:
Let's update the view tool to print "unknown core" which will be much more clear than this giant number.

Reply:
Done.